README.md

```diff
@@ -1066,6 +1066,7 @@ Table of Contents
   * [Bricks Note Calculator](https://free.getbricks.app/) - a note-taking app (PWA) with a powerful built-in multiline calculator.
   * [Code Time](https://www.software.com/code-time) - an extension for time-tracking and coding metrics in VS Code, Atom, IntelliJ, Sublime Text, and more.
   * [ConfigCat](https://configcat.com) - Cross-platform feature flag service. SDKs for all major languages. Free plan up to 10 flags, 2 environments, 1 product and 5 Million requests per month. Unlimited user seats. Students get 100 flags and 100 Million requests per month for free.
+  * [datelist.io](https://datelist.io) - Online booking system for your website
```

Comment:
is there an english website? We're unable to verify anything but english sites.

Reply:
Looked at the website. Current offers on the free plan:
Up to 5 reservations per month
1 Calendar
Email notifications
Widget for your website
Dedicated booking page